.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -37,26 +37,40 @@ members/contributors which may be interested in your PR.
  If you know how to use git blame, that is the easiest way, otherwise, here is a rough guide of **who to tag**.
  Please tag fewer than 3 people.
 
- albert, bert, XLM: @LysandreJik
- GPT2: @LysandreJik, @patrickvonplaten
- tokenizers: @mfuntowicz
- Trainer: @sgugger
- Benchmarks: @patrickvonplaten
- Model Cards: @julien-c
- examples/distillation: @VictorSanh
+Models:
+
+ albert, bert, xlm: @LysandreJik
+ blenderbot, bart, marian, pegasus, encoderdecoder, longformer, reformer, t5, transfoxl, xlnet: @patrickvonplaten
+ fsmt: @stas00
+ funnel: @sgugger
+ gpt2: @patrickvonplaten, @LysandreJik
+ rag: @patrickvonplaten, @lhoestq
+ tensorflow: @jplu
+
+Library:
+
+ benchmarks: @patrickvonplaten
+ deepspeed: @stas00
+ ray/raytune: @richardliaw, @amogkam
+ text generation: @patrickvonplaten
+ tokenizers: @n1t0
+ trainer: @sgugger
+
+Documentation: @sgugger
+
+HF projects:
+
  nlp datasets: [different repo](https://github.com/huggingface/nlp)
  rust tokenizers: [different repo](https://github.com/huggingface/tokenizers)
- Text Generation: @patrickvonplaten, @TevenLeScao
- Blenderbot, Bart, Marian, Pegasus: @patrickvonplaten
- T5: @patrickvonplaten
- Rag: @patrickvonplaten, @lhoestq
- EncoderDecoder: @patrickvonplaten
- Longformer, Reformer: @patrickvonplaten
- TransfoXL, XLNet: @TevenLeScao, @patrickvonplaten
- examples/seq2seq: @patil-suraj
- examples/bert-loses-patience: @JetRunner
- tensorflow: @jplu
+
+Maintained examples:
+
+ examples/seq2seq: @patil-suraj, @sgugger
  examples/token-classification: @stefan-it
- documentation: @sgugger
- FSMT: @stas00
+
+Research Projects:
+
+ examples/research_projects/bert-loses-patience: @JetRunner
+ examples/research_projects/distillation: @VictorSanh
```

Comment:
I was more thinking of:
```
Examples:

    maintained examples (not research project, not legacy) @sgugger @patil-suraj 
    research_projects/bert-loses-patience: @JetRunner
    research_projects/distillation: @VictorSanh
```
The new token classification example was written by me, so there is no need to tag stefan anymore on it.

Reply:
Done